.mailmap

```diff
@@ -0,0 +1,5 @@
+Lee Clemens <java@leeclemens.net>
+Serg G. Brester <info@sebres.de>
+Serg G. Brester <serg.brester@sebres.de>
+Serg G. Brester <sergey.brester@W7-DEHBG0189.wincor-nixdorf.com>
```

Comment:
please remove this one, because not real e-mail address (I don't know at all where I have used this one!)